### ISSUE TITLE
Some questions about validation 

### ISSUE THREAD

**Issue by yangH8** [shahaf1313/ProCST#4]
We want to test the model on the validation set in 'calculte_validation_accuracy' function.  But we has been trained 5 epochs, it's still not into the validation stage. How can we quickly validate the model?

**Comment by shahaf1313** [comment]
Hey Yang,
Validation on the target set is enabled only at the last scale (e.g. n=N), because till this stage we do not handle full resolution images. If you wish the code to validate please train the model up to the last scale. 
Additionally, you can change the behavior of the code to support such a feature. I assume that the files that needs to to be changed are core/training.py and  data_handlers/cityscapes_dataset.py.
Shahaf.

**Comment by yangH8** [comment]
How many scales are there?

**Comment by shahaf1313** [comment]
3 total. It is configurable of course.
